### PR TITLE
chore: comment gateway environment variables

### PR DESCRIPTION
**Pull request by m1sterc001guy**
Just adding some comments. Likely helpful for deployments for figuring out how to configure the gateway.

### CODE REVIEW

File: gateway/ln-gateway/src/envs.rs [hunk 1]
```diff
@@ -1,50 +1,75 @@
...
-pub const FM_CLN_EXTENSION_LISTEN_ADDRESS_ENV: &str = "FM_CLN_EXTENSION_LISTEN_ADDRESS";
-
-// Env variable to TODO
+// Environment variable that specifies the directory of the gateway's database.
 pub const FM_GATEWAY_DATA_DIR_ENV: &str = "FM_GATEWAY_DATA_DIR";
 
-// Env variable to TODO
+// Environment variable that specifies the address the gateway's HTTP server
+// should listen on.
 pub const FM_GATEWAY_LISTEN_ADDR_ENV: &str = "FM_GATEWAY_LISTEN_ADDR";
```
Comment (elsirion) [inline gateway/ln-gateway/src/envs.rs:6]: Since you are already touching the comments, can you make them doc comments please (three slashes)?

Comment (m1sterc001guy) [reply gateway/ln-gateway/src/envs.rs]: Done

File: gateway/ln-gateway/src/envs.rs [hunk 2]
```diff
@@ -1,50 +1,77 @@
...
+pub const FM_GATEWAY_LIGHTNING_ADDR_ENV: &str = "FM_GATEWAY_LIGHTNING_ADDR";
+
+/// Environment variable that specifies the mnemonic that the gateway should use
+/// for ecash and the LDK Node should use for onchain funds. If not set, a
+/// mnemonic will be generated. This environment variable can be used for
+/// recovering from an existing mnemonic.
 pub const FM_GATEWAY_MNEMONIC_ENV: &str = "FM_GATEWAY_MNEMONIC";
 
+/// Environment variable that specifies the "module mode" the gateway should run
+/// in. Options are "LNv1", "LNv2", or "All". It is not recommended to run "all"
```
Comment (bradleystachurski) [inline gateway/ln-gateway/src/envs.rs:69]: [nit] Is "All" case sensitive? If so, then it may be more clear to `s/all/All`

**Review by bradleystachurski — DISMISSED** [review]
Thanks for adding docs!

File: gateway/ln-gateway/src/envs.rs (hunk 2 above)
Comment (m1sterc001guy) [reply gateway/ln-gateway/src/envs.rs]: not case sensitive but good to be consistent. Fixed in the rebase :)

**Review by bradleystachurski — APPROVED** [review]

**Pull request merged**